.github/workflows/publish-docs.yaml

```diff
@@ -1,10 +1,22 @@
-name: Publish Documentation
+name: Publish Documentation (Manual)
 
+# Manual workflow for republishing specific documentation versions
+# For new release docs, use the release workflow instead
 on:
-  push:
-    branches:
-      - main
   workflow_dispatch:
+    inputs:
+      tag:
+        description: "Version tag to republish (e.g., 1.0.0, 1.1.0)"
+        required: true
+      branch:
+        description: "Branch to get documentation content from (leave empty for current branch)"
+        required: false
+        default: ""
```

Comment:
should you really be able to re release from any random branch. Don't you want a pattern like `main` or `hotfix/*`

Reply:
I like the idea of hotfix